.vib/nginx/goss/goss.yaml

```diff
@@ -0,0 +1,32 @@
+file:
+  /opt/bitnami/nginx/conf/nginx.conf:
```

Comment:
It may be interesting to check some of the configuration passed as `runtime_params` is applied here. Something like:

```
contains:
  - /listen\s+8080/
```

Reply:
I agree, as it is now this check is a bit redundant given that we also have the `nginx-config-files`. In it, we are implicitly checking the `nginx.conf` file exists.